src/modules/sv_detection_methods/analyze_split_read_method.cpp

```diff
@@ -29,6 +29,12 @@ void retrieve_aligned_segments(std::string const & sa_string, std::vector<Aligne
         if (fields.size() == 6)
         {
             std::string ref_name = fields[0];
+            // As the chromosome names can differ between "1" and "chr1", we would distinguish same SVs from different
+            // input files if the chromosome naming is different. Thus we remove the "chr" prefix.
+            if (ref_name.rfind("chr", 0) == 0) { // pos=0 limits the search to the prefix
```

Comment:
Interesting way of implementing this 😏 
It's fine, however I think that using `ref_name.starts_with()` would be more intuitive ([Link](https://en.cppreference.com/w/cpp/string/basic_string/starts_with)).